addons/website_forum/static/src/js/website_forum.js

```diff
@@ -243,7 +260,8 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         };
         if (session.is_website_user) {
             notifOptions.title = _t("Access Denied");
-            notifOptions.message = _t("Sorry you must be logged in to perform this action");
+            notifOptions.message =  Markup`
+            <p>Oh no! Please <a href='/web/login'>sign in</a> to vote</p>`;
```

Comment:
Why multi-line ?

Reply:
same, and actually the same for the empty `style=""` above